UIKitDZ/UIKitDZ/SettingsViewController/SettingsViewController.swift

```diff
@@ -0,0 +1,204 @@
+//
+//  SettingsViewController.swift
+//  UIKitDZ
+//
+//  Created by Анна Сычева on 04.10.2022.
+//
+
+import UIKit
+/// Экран настройки отображения текста
+final class SettingsViewController: UIViewController {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let smallASystemName = "textformat.size.smaller"
+        static let largeASystemName = "textformat.size.larger"
+        static let fontLabelText = "Выберите шрифт"
+        static let segmentControlRegular = "Regular"
+        static let segmentControlBold = "Bold"
+    }
+    
+    // MARK: - Public properties
+    
+    weak var delegate: ReaderViewControllerDelegate?
+    
+    // MARK: - Private properties
+    
+    private lazy var fontLabel = makeFontLabel()
+    private lazy var fontPicker = makeFontPicker()
+    private lazy var fontSizeSlider = makeFontSizeSlider()
+    private lazy var smallA = makeAImageView(name: Constants.smallASystemName, xCoordinate: 20)
+    private lazy var largeA = makeAImageView(name: Constants.largeASystemName, xCoordinate: 374)
+    private lazy var boldSegmentControl = makeBoldSegmentControl()
+    private lazy var darkModeSwitch = makeDarkModeSwitch()
+    private lazy var colors: [UIColor] = [.systemPink, .systemBlue, .systemBrown, .systemOrange]
+    private lazy var fonts: [String] = [String]()
+    private lazy var selectFont = ""
```

Comment:
```suggestion
    private lazy var selectFont = ""
```
а для чего нам ленивая инициализация например здесь или на 38 строчке?